app/src/main/kotlin/org/eclipse/kuksa/companion/feature/connection/view/VerticalConnectionStatusView.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.eclipse.kuksa.companion.feature.connection.view
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.width
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.constraintlayout.compose.ConstraintLayout
+import org.eclipse.kuksa.companion.R
+import org.eclipse.kuksa.companion.extension.isVisible
+import org.eclipse.kuksa.companion.feature.connection.viewModel.ConnectionStatusViewModel
+
+@Composable
+fun VerticalConnectionStatusView(
```

Comment:
In general a good idea, however imo not applicable for our use case the since we already splitted everything up by feature. We have 7 features, and by splitting this stuff up again by landscape portrait we have basically 14 packages (landscape, portrait) in which a single view is put plus we have to traverse the folder structure through these 14 folders when navigating from file to file. 

It would have worked to split up by orientation instead of by feature

Therefore I prefer to keep the stuff together and not introduce packages for single files. 

> [...] makes the pattern clearer to use separate compose files for these cases

I don't see any clarity added by a landscape / portrait package when the Horizontal- / Vertical- / Adaptive- Views can reside next to each other to be honest.

Reply:
OK for me